<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionView</class>
 <widget class="QMainWindow" name="SectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Section Viewer</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="fmeMap">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>450</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="vlyt">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="lytMap"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Coordinate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledCoord">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Looking Towards</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledFacing">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="lytScale">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>218</width>
     <height>240</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QWidget" name="wdgLegend" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="lytLegend"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockIdent">
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>150</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Identify Results</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="lytIdentify">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QTreeWidget" name="identTree">
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>110</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>110</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Measure Tool</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledSegLen"/>
     </item>
     <item row="0" column="0" rowspan="4">
      <widget class="QListWidget" name="lstSeg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ledTotalLen"/>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionZoom_in">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionZoomIn.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionZoomIn.svg</iconset>
   </property>
   <property name="text">
    <string>zoom_in</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionZoomOut.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionZoomOut.svg</iconset>
   </property>
   <property name="text">
    <string>zoom_out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionTouch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionTouch.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionTouch.svg</iconset>
   </property>
   <property name="text">
    <string>touch</string>
   </property>
   <property name="toolTip">
    <string>Touch Zoom and Pan</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionPan.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionPan.svg</iconset>
   </property>
   <property name="text">
    <string>pan</string>
   </property>
   <property name="toolTip">
    <string>Pan Map</string>
   </property>
  </action>
  <action name="actionGenerateSection">
   <property name="text">
    <string>Generate Section</string>
   </property>
   <property name="toolTip">
    <string>Generate a new cross section</string>
   </property>
  </action>
  <action name="actionIdentify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionIdentify.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionIdentify.svg</iconset>
   </property>
   <property name="text">
    <string>Identify</string>
   </property>
  </action>
  <action name="actionMeasureTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionMeasure.png</normaloff>:/plugins/QDriller/Icons/icons/mActionMeasure.png</iconset>
   </property>
   <property name="text">
    <string>MeasureTool</string>
   </property>
  </action>
  <action name="actionMeasureArea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionMeasureArea.png</normaloff>:/plugins/QDriller/Icons/icons/mActionMeasureArea.png</iconset>
   </property>
   <property name="text">
    <string>Measure Area</string>
   </property>
  </action>
  <action name="actionSelectTool">
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select Features</string>
   </property>
  </action>
  <action name="actionPolygonSelect">
   <property name="text">
    <string>Polygon Select</string>
   </property>
   <property name="toolTip">
    <string>Select features using polygon</string>
   </property>
  </action>
  <action name="actionManageSections">
   <property name="text">
    <string>Manage Sections</string>
   </property>
   <property name="toolTip">
    <string>Manage available sections</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/transformed.png</normaloff>:/plugins/QDriller/Icons/icons/transformed.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>add grid to canvas</string>
   </property>
  </action>
  <action name="actionSaveAsImage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionSaveMapAsImage.png</normaloff>:/plugins/QDriller/Icons/icons/mActionSaveMapAsImage.png</iconset>
   </property>
   <property name="text">
    <string>save as Image</string>
   </property>
   <property name="toolTip">
    <string>save canvas as image</string>
   </property>
  </action>
  <action name="actionAddLayer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionAddOgrLayer.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionAddOgrLayer.svg</iconset>
   </property>
   <property name="text">
    <string>addLayer</string>
   </property>
   <property name="toolTip">
    <string>Add Blank Vector Layer to Section</string>
   </property>
  </action>
  <action name="actionToggleEditing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mIconEditable.png</normaloff>:/plugins/QDriller/Icons/icons/mIconEditable.png</iconset>
   </property>
   <property name="text">
    <string>ToggleEditing</string>
   </property>
   <property name="toolTip">
    <string>Toggle Editing of layer</string>
   </property>
  </action>
  <action name="actionAddPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionCapturePoint.png</normaloff>:/plugins/QDriller/Icons/icons/mActionCapturePoint.png</iconset>
   </property>
   <property name="text">
    <string>addPoint</string>
   </property>
   <property name="toolTip">
    <string>Add Point Feature</string>
   </property>
  </action>
  <action name="actionAddLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionCaptureLine.png</normaloff>:/plugins/QDriller/Icons/icons/mActionCaptureLine.png</iconset>
   </property>
   <property name="text">
    <string>addLine</string>
   </property>
   <property name="toolTip">
    <string>Add Line Feature</string>
   </property>
  </action>
  <action name="actionAddPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionCapturePolygon.png</normaloff>:/plugins/QDriller/Icons/icons/mActionCapturePolygon.png</iconset>
   </property>
   <property name="text">
    <string>addPolygon</string>
   </property>
   <property name="toolTip">
    <string>Add Polygon Feature</string>
   </property>
  </action>
  <action name="actionSaveEdits">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionSaveAllEdits.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionSaveAllEdits.svg</iconset>
   </property>
   <property name="text">
    <string>Save Edits</string>
   </property>
   <property name="toolTip">
    <string>Save changes to current layer</string>
   </property>
  </action>
  <action name="actionRollBackEdits">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionRollbackAllEdits.svg</normaloff>:/plugins/QDriller/Icons/icons/mActionRollbackAllEdits.svg</iconset>
   </property>
   <property name="text">
    <string>RollBack Edits</string>
   </property>
   <property name="toolTip">
    <string>Rollback changes to current layer</string>
   </property>
  </action>
  <action name="actionMoveFeature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionMoveFeature.png</normaloff>:/plugins/QDriller/Icons/icons/mActionMoveFeature.png</iconset>
   </property>
   <property name="text">
    <string>move Feature</string>
   </property>
   <property name="toolTip">
    <string>Move Feature Location</string>
   </property>
  </action>
  <action name="actionNodeTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/QDriller/Icons/icons/mActionNodeTool.png</normaloff>:/plugins/QDriller/Icons/icons/mActionNodeTool.png</iconset>
   </property>
   <property name="text">
    <string>nodeTool</string>
   </property>
   <property name="toolTip">
    <string>Edit Nodes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
